sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -247,11 +247,6 @@ def start_k8s_software(self):
             minikube delete || true
             minikube start --driver=none --extra-config=apiserver.service-node-port-range=1-65535
             chmod 777 -R $MINIKUBE_HOME
-            # [ -z "$KUBECONFIG" ] || exit
-            # ABS_KUBECONFIG_DIR=$(dirname $(realpath -m $KUBECONFIG))
-            # ABS_MINIKUBE_DIR=$(realpath -m ~/.minikube)
-            # cp -r $ABS_MINIKUBE_DIR $ABS_KUBECONFIG_DIR
-            # sed -ri "s/${{ABS_MINIKUBE_DIR////\\\\/}}/${{ABS_KUBECONFIG_DIR////\\\\/}}\\/.minikube/g" $ABS_KUBECONFIG_DIR/config
```

Comment:
How is this change related to the ignoring failure uploading existing image?

Reply:
moved out to separate commit